src/utils/addDockerSettingsToEnv.ts

```diff
@@ -0,0 +1,28 @@
+
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { workspace } from 'vscode';
+import { configPrefix } from '../constants';
+import { ext } from '../extensionVariables';
+
+export function addDockerSettingsToEnv(env: {}): void {
+    addDockerSettingToEnv("host", 'DOCKER_HOST', env);
+    addDockerSettingToEnv("certPath", 'DOCKER_CERT_PATH', env);
+    addDockerSettingToEnv("tlsVerify", 'DOCKER_TLS_VERIFY', env);
+    addDockerSettingToEnv("machineName", 'DOCKER_MACHINE_NAME', env);
+}
+
+function addDockerSettingToEnv(settingKey: string, envVar: string, env: {}): void {
```

Comment:
So VS Code settings are respected above env settings that may currently exist?  For instance, `DOCKER_HOST` would get overridden if I had the `host` setting in my project?

Reply:
Yeah, that's what users requested. They might have the environment variable set on their machine, but they want VS Code settings to override that on a per-workspace level

Reply:
It definitely makes sense, but I was wondering if we should alert users in the output that their env variable is getting overwritten.  I could totally see myself forgetting I had a setting and wondering why my environment variables weren't being used 😅

Reply:
Added a warning